addons/web/static/src/core/datetime/datetime_picker.scss

```diff
@@ -21,80 +21,83 @@
             var(--DateTimePicker__Cell-border-radius-end)
             var(--DateTimePicker__Cell-border-radius-end)
             var(--DateTimePicker__Cell-border-radius-start);
+        position: relative;
```

Comment:
Convention: position directives should be placed before border (https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html?highlight=css#properties-order)